lib/sqlalchemy/dialects/mssql/__init__.py

```diff
@@ -39,11 +38,11 @@
 from .base import TIME
 from .base import TIMESTAMP
 from .base import TINYINT
-from .base import try_cast
 from .base import UNIQUEIDENTIFIER
 from .base import VARBINARY
 from .base import VARCHAR
 from .base import XML
+from ...sql._elements_constructors import try_cast
```

Comment:
**mike bayer** (zzzeek) wrote:

this can be imported as 

    from ...sql import try_cast as try_cast

that is let's not reach into a private module...

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4605